backend/api/v1/auth_service.go

```diff
@@ -416,11 +420,12 @@ func (*AuthService) Logout(ctx context.Context, _ *v1pb.LogoutRequest) (*emptypb
 }
 
 func (s *AuthService) getUserWithLoginRequestOfBytebase(ctx context.Context, request *v1pb.LoginRequest) (*store.UserMessage, error) {
-	emptyIDP := ""
 	user, err := s.store.GetUser(ctx, &store.FindUserMessage{
```

Comment:
Maybe we can do a ListUsers() first. And only backfill when there's only one user.

Reply:
Done